test/WebJobs.Script.Tests.Integration/Management/FunctionsSyncManagerTests.cs

```diff
@@ -311,6 +311,21 @@ public async Task TrySyncTriggers_LocalEnvironment_ReturnsFalse()
             }
         }
 
+        [Fact]
+        public async Task TrySyncTriggers_ManagedAppEnv_WithNo_AzureWebJobsStorage_ReturnsTrue()
+        {
+            using (var env = new TestScopedEnvironmentVariable(_vars))
+            {
+                _mockEnvironment.Setup(p => p.GetEnvironmentVariable(EnvironmentSettingNames.ManagedEnvironment)).Returns("true");
+                _mockEnvironment.Setup(p => p.GetEnvironmentVariable("FUNCTIONS_API_SERVER")).Returns("https://appname.azurewebsites.net");
+                _mockEnvironment.Setup(p => p.GetEnvironmentVariable("CONTAINER_APP_NAME")).Returns("appname");
+                _mockEnvironment.Setup(p => p.GetEnvironmentVariable("CONTAINER_APP_NAMESPACE")).Returns("appns");
+                _mockEnvironment.Setup(p => p.GetEnvironmentVariable("CONTAINER_APP_REVISION")).Returns("appname--r1");
+                var result = await _functionsSyncManager.TrySyncTriggersAsync(isBackgroundSync: true);
+                Assert.True(result.Success);
```

Comment:
This assert doesn't seem sufficient. E.g. If everything is short circuited [here](https://github.com/Azure/azure-functions-host/blob/46194b9893def8e31a76a69e6dfbfeac5530834f/src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs#L122), your test would still pass, right?